mesa/agent.py

```diff
@@ -606,6 +606,31 @@ def do(self, method: Callable | str, *args, **kwargs) -> GroupBy:
 
         return self
 
+    def count(self) -> dict[Any, int]:
+        """
+        Return the count of agents in each group.
+
+        Returns:
+            dict: A dictionary mapping group names to the number of agents in each group.
+        """
+        return {k: len(v) for k, v in self.groups.items()}
+
+    def agg(self, attr_name: str, func: Callable) -> dict[Any, Any]:
```

Comment:
Shouldn't the return be `dict[Hashable, Any]`?

Reply:
Good call, you're right. Implemented in 817b532.